app/views/users/show.html.erb

```diff
@@ -1,18 +1,52 @@
-<h1>
-  <%= @user.username.upcase %>
-</h1>
-
-<h2> Average rating </h2>
-<% average_rating = 0 %>
-<% counter = 0 %>
-
-<% Rating.all.each do |a_rating| %>
-  <% if a_rating.gradee == @user %>
-    <% average_rating = average_rating + a_rating.grade %>
-    <% counter = counter + 1%>
-  <% end %>
-<% end %>
+<div class="row mb-4">
+  <div class="col-md-6 offset-md-3">
+    <h1 class="display-4">
+      <%= "#{@user.username.capitalize()}'s rating" %>
```

Comment:
If a method has no arguments it's much more common to leave out the parenthesis.

```suggestion
      <%= "#{@user.username.capitalize}'s rating" %>
```